### ISSUE TITLE
Partner promos

### ISSUE THREAD

**Issue by celrae** [energy7/datastore#10]
I keep trying to click on the partner logo/text blocks -- I don't know if people will do that, but perhaps we need to make that clickable to the partner's collection page. Maybe just a note to pass on to back-end development phase? Logging for future if not for today.

**Comment by energy7** [comment]
@celrae Yeah, I wasn't sure if you would want to direct traffic off-site or if they'd all have internal collections where you could send clicks. For now they're just images of the biggest names you can drop :)
